hassio/snapshots/hassio-snapshot.html

```diff
@@ -139,7 +177,7 @@ <h2>[[computeName(snapshot)]]</h2>
     return type === 'full';
   }
 
-  partialRestoreTapped() {
+  partialRestoreClicked() {
```

Comment:
Prefix functions with `_` as a convention to show that they are private and only used inside your element.